library/public/loader.h

```diff
@@ -0,0 +1,94 @@
+/**
+ * @class   loader
```

Comment:
Should this be renamed `renderer` ? 

It seems like:
* addFile read from disk (or memory) and represent is an "f3d object"
`loadFile()` is in-fact `render(objectIdx)`

Reply:
Im afraid it is not. loadFile do not render anything. window::render() does, as well as interactor::start()

Reply:
Since the loader seems to a convenience class for selecting files, I would suggest to keep it specific to the application and not expose it in the public API.

Application integrating this library will likely have their own UI library providing a tailored way of organizing the files that is addressing their domain specific use case.

Reply:
The loader load the file, I think this is easy to understand for users.